ephemeris/sleep.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import requests
+import sys
+import time
+
+from argparse import ArgumentParser
+
+from .common_parser import get_common_args
+from .shed_install import setup_global_logger, _disable_external_library_logging
+
+
+def _parse_cli_options():
+    """
+    Parse command line options, returning `parse_args` from `ArgumentParser`.
+    """
+    parent = get_common_args(login_required=False)
+    parser = ArgumentParser(parents=[parent], usage="usage: python %(prog)s <options>")
```

Comment:
Can we add here a small description about what this tool is doing or make the tool name a little bit more descriptive?